src/services/azureApplicationInsights.js

```diff
@@ -43,9 +43,14 @@ const createTelemetryService = () => {
     });
 
     appInsights.loadAppInsights();
+    if (process.env.NODE_ENV === "development") {
+      // Normally metrics are sent as batches. In development mode, send immediately.
```

Comment:
oh interesting! useful...

Reply:
is there a super easy way to send immediately for staging as well? no problem if not, just seems potentially useful if we're debugging on staging and watching the metrics in real time

Reply:
Sure! I added a domain check so on staging it will be sent immediately.